components/forms/RadioButton/stories.react.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withKnobs, boolean } from '@storybook/addon-knobs'
+import { withState } from '@dump247/storybook-state'
+import Radio from './react'
+
+storiesOf('Forms/Radio', module)
+  .addDecorator(withKnobs)
+  .add(
+    'checked with a click event',
+    withState({ checked: true })(({ store }) => (
+      <React.Fragment>
+      <Radio
+        checked={store.state.checked}
+        onChange={() => store.set({ checked: !store.state.checked })}
+        name={store.state.name}
+      />
+      </React.Fragment>
+    )),
+  )
+
+  .add('fully controlled', () => {
```

Comment:
This story isn't needed, as it is already demonstrated fully controlled. Check out this post to understand controlled vs uncontrolled https://reactjs.org/docs/forms.html#controlled-components